### DISCUSSION TITLE
抱歉，请联系机房域名过白 怎么处理

### DISCUSSION THREAD

**Discussion by lin1993** [natfrp/wiki#367] [使用问题]
换了很多节点都不行啊，实名认证也过了，啥问题呀，机房客服我哪知道谁

抱歉，请联系机房域名过白
Sorry,the site now can not be accessed.
网站可能因以下原因之一而导致无法访问：
域名未备案、网站过期被系统关闭.
请联系机房客服.

**Comment by fakeboboliu** [comment]
请看文档 https://doc.natfrp.com/faq/site-inaccessible.html
